scripts/upgrade.sh.in

```diff
@@ -6,6 +6,7 @@ set -o pipefail
 VERBOSE=${VERBOSE:-0}
 LOGFILE=/var/log/metalk8s/upgrade.log
 DRY_RUN=0
+SALTENV=${SALTENV:-}
```

Comment:
maybe this should also be a CLI argument?

Reply:
at least, it should appear in docs somewhere, don't you think?

Reply:
Hmm for me no, it's like salt-call we do not want the user to change it, but it's doable if he really wants

Reply:
OK, it'll be easier once we have those as discoverable flags with the Python CLI.